scripts/generate_error_code.hs

```diff
@@ -93,10 +93,9 @@ invalidInputErrors = errorsEnumerate 100
     :? ("There are no available proposals to flush.", NoArg)
   , "not_delegate"
     :? ("The sender has not been delegated the control of the required tokens.", NoArg)
-  , "fail_decision_lambda"
+  , "fail_decision_callback"
     :? ("Executing the proposal's decision lambda results in failure.", NoArg)
```

Comment:
I suppose that "lambda" should be removed from the description as well.

Reply:
Sorry about this. Have renamed lambda everywhere now.